js/logo.js

```diff
@@ -1251,6 +1234,22 @@ class Logo {
         tur.singer.whichNoteToCount = 1;
         tur.singer.moveable = false;
 
+        tur.singer.vibratoIntensity = [];
```

Comment:
I wonder if all of these inits could be moved to a method inside singer?

Reply:
Yes I'll move the entire initTurtle method to Turtle, but I'd need to transport all the variables from Logo first